recipe/build.sh

```diff
@@ -26,7 +26,7 @@ CPPFLAGS="-I$PREFIX/include" LDFLAGS="-L$PREFIX/lib" \
             --with-python \
             $PGFLAG
 
-make
+make -s
```

Comment:
Can you do something along these lines (untested):

```shell
if [[ $(uname) == Darwin ]]; then
 OPTS="-s"  # Silence log to avoid Travis-CI 4 MB limit,
else
 OPTS=""
fi

make
make $OPTS
```